bodhi/tests/server/test_models.py

```diff
@@ -626,6 +626,81 @@ def test_days_to_stable_not_meets_testing_requirements_with_date_testing(self):
 
         eq_(update.days_to_stable, 3)
 
+    @mock.patch.dict('bodhi.server.config.config', {'ci.required': True})
```

Comment:
Be sure to use a string "tRuE" (or whatever weird capitalization you like) here, and elsewhere in this module.